beetmoverscript/constants.py

```diff
@@ -23,6 +23,7 @@
 TEMPLATE_KEY_PLATFORMS = defaultdict(lambda: "firefox", {
     "android-api-15": "fennec",
     "android-x86": "fennecx86",
+    "android-5-0-aarch64": "fennec",
```

Comment:
looking at:
https://github.com/mozilla-releng/beetmoverscript/blob/549e60a5fdc8f2a967624198126a5c87168aeb4b/beetmoverscript/utils.py#L67
https://public-artifacts.taskcluster.net/QztCdLbzSy2YAR6CNRzhOA/0/public/logs/live_backing.log

makes me think this should be "android-aarch64"

because of:

[task 2017-05-26T15:50:53.970065Z] 15:50:53     INFO - Dumping config to /home/worker/artifacts/balrog_props.json.
[task 2017-05-26T15:50:53.970251Z] 15:50:53    DEBUG - mkdir_p: /home/worker/artifacts Already exists.
[task 2017-05-26T15:50:53.971081Z] 15:50:53     INFO - {'properties': {'appName': 'Fennec',
[task 2017-05-26T15:50:53.971253Z] 15:50:53     INFO -                 'appVersion': '55.0a1',
[task 2017-05-26T15:50:53.971410Z] 15:50:53     INFO -                 'branch': 'mozilla-central',
[task 2017-05-26T15:50:53.971556Z] 15:50:53     INFO -                 u'buildername': u'',
[task 2017-05-26T15:50:53.971703Z] 15:50:53     INFO -                 'buildid': '20170526150310',
[task 2017-05-26T15:50:53.971854Z] 15:50:53     INFO -                 'builduid': '7450f879c0c0410db4866704cad502ba',
[task 2017-05-26T15:50:53.972041Z] 15:50:53     INFO -                 'completeMarHash': u'9ed12dc292aa17f28b46a1f3a8b6e5ef81b951f26cb4f03dcfe28e9f7392ac56d15d376bc37dd01c4ccb6c106d0d952d74ab16c720377b8452ffd110c5dc12bc',
[task 2017-05-26T15:50:53.972165Z] 15:50:53     INFO -                 'completeMarSize': u'50011071',
[task 2017-05-26T15:50:53.972307Z] 15:50:53     INFO -                 'hashType': 'sha512',
[task 2017-05-26T15:50:53.972460Z] 15:50:53     INFO -                 'sourcestamp': '9104506c0a5eaac89b6c22c804c83ccee7d6e662',
[task 2017-05-26T15:50:53.972601Z] 15:50:53     INFO -                 'stage_platform': 'android-aarch64'}}